lib/bloom_trade_client/responses/base_response.rb

```diff
@@ -3,13 +3,23 @@ class BaseResponse
     include APIClientBase::Response.module
 
     attribute :body, String, lazy: true, default: :default_body
+    attribute :errors, Hash, lazy: true, default: :default_errors
 
     private
 
     def default_body
       JSON.parse(raw_response.body).with_indifferent_access
     end
 
+    def default_errors
+      errors = if body.is_a? Hash
```

Comment:
When will body not be a hash?

Reply:
If `.body` was supplied by a `String`. I'll probably make it strict.